runners/katacoda/index.ts

```diff
@@ -399,6 +399,34 @@ export class Katacoda extends Runner {
         return null;
     }
 
+    runExecuteCommand(runCommand: RunCommand) : RunResult {
+        let terminal = (runCommand.command.parameters.length > 2 && runCommand.command.parameters[2].asynchronous) 
+            ? this.getTerminal("executeCommand"+runCommand.stepIndex) 
+            : undefined;
+        
+        let filepath;
+        let changeDir = false;
+        if(runCommand.command.parameters.length > 2 && runCommand.command.parameters[2].dir){
+            filepath = runCommand.command.parameters[2].asynchronous 
+                ? path.join(this.getVariable(this.workspaceDirectory), runCommand.command.parameters[2].dir).replace(/\\/g, "/")
+                : runCommand.command.parameters[2].dir;
+            changeDir = true;
+        }
+
+        let bashCommand = {
+            "name" : runCommand.command.parameters[1],
+            "changeDir" : changeDir,
```

Comment:
Can't we use the changeCurrentDir method in this case? At least you have to change the currentDir variable when changing the directory.

Reply:
I need also to use interrupt for some cd commands. I can add it to the changeCurrentDir method or i will just leave it like that and currentDir will be updated now.